lib/validate.ts

```diff
@@ -311,6 +312,7 @@ export function getValidators(
             new UnitTypeValidator(error, componentSet),
             new LocalizationValidator(error, componentSet, filePaths, getFileContent),
             new PackageValidator(error, componentSet, filePaths, getFileSize),
+            new CustomStylesValidator(error, componentSet, filePaths),
```

Comment:
For now I added the validator to all versions. The advantage of this is that the validator checks all the uploads for files in the `styles/customStyles` directory.